tests/tests.h

```diff
@@ -187,6 +189,40 @@ namespace Testing
 
 
 
+// Get a uniformly distributed random value between min and max
+template<type T=double>
+T random_value(const T &min, const T &max)
```

Comment:
i still suggest we have a single function

```
template<type T=double>
inline
 T random_value(const T &max=1., const T &min=0.)
{
   //...
}
```

which will cover all usage cases.

Reply:
95% of the use cases only require the maximum value to be specified. Having one single function requires you to write in all 95% of the cases `random_value(0, max)`.

This follows the same "rule" of the `range()` function in python (and in fact that' also how I use it:`random_value(size-1)` is faster than `random_value(0,size-1)` for `decltype(size)=unsigned int`...

what do other people think?

Reply:
@luca-heltai 

> Having one single function requires you to write in all 95% of the cases random_value(0, max).

pease read again my comment above, specifically pay attention to the arguments, where `max` comes first. As I said, in the same 95% of cases you will still be able to do `random_value(max)`.

Reply:
Don't you find it counter-intuitive to have max first and min second? I'll do as you wish, but I disagree with your view...

Reply:
I think that you could leave this as it is.

Reply:
I don’t if reflects the usage cases, (i) both defaults are good (ii) max needs adjustment (iii) max and min needs adjustment

Reply:
To me, `(min,max)` would be the intuitive use of such a function.

Reply:
It would seem to me that a fair compromise might be to remove the version of the function that takes a single argument. @luca-heltai As far as I could tell, this would require only one amendment in terms of use cases: `random_value<double>(.9)` -> `0.9*random_value<double>()` in ` tests/bits/joa_1.cc`

Reply:
> a fair compromise might be to remove the version of the function that takes a single argument

i am fine with that.